README.md

```diff
@@ -49,16 +49,23 @@ module Main where
 
 import Build_doctests (flags, pkgs, module_sources)
 import Data.Foldable (traverse_)
+import System.Environment.Compat (unsetEnv)
 import Test.DocTest (doctest)
 
 main :: IO ()
 main = do
     traverse_ putStrLn args -- optionally print arguments
+    unsetEnv "GHC_ENVIRONMENT" -- see 'Notes'; you may not need this
     doctest args
   where
     args = flags ++ pkgs ++ module_sources
 ```
 
+(The `System.Environment.Compat` module is from the `base-compat`
+package. That's already in the transitive closure of `doctest`'s
+dependencies. If you don't need to support GHC 7.6 or earlier, you can
```

Comment:
I'd like to phrase that using *GHC-7.8*, i.e. mention the first GHC which doesn't need `.Compat`. Something like *if you support only a subset of `GHC >= 7.8, base >= 4.7` compilers, then you can use `System.Environment` directly*. But I don't know how to say that in good English.

Reply:
This phrasing better?